server/@types/express/index.d.ts

```diff
@@ -9,6 +9,8 @@ declare module 'express-session' {
     returnTo: string
     nowInMinutes: number
     user: ServiceUser
+    // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+    data: any
```

Comment:
Is there a less generic name you can give this? What is it?

Reply:
This is the express session - I needed to store the locations on it so that it would survive a redirect.
`    req.session.data = {
      activityLocations,
    }`
But i didnt want to restrict what kind of data you could put on the session.
Was hoping you could advise if theres a better approach